apm/config.go

```diff
@@ -90,14 +96,14 @@ func (c *MonitorConfig) window() map[eventKey]*eventRecord {
 	for _, db := range c.Databases {
 		for _, coll := range c.Collections {
 			for _, cmd := range c.Commands {
-				out[eventKey{dbName: db, collName: coll, cmdName: cmd}] = &eventRecord{}
+				out[eventKey{dbName: db, collName: coll, cmdName: cmd}] = &eventRecord{Tags: map[string]int64{}}
```

Comment:
Is there a reason we need to initialize this to an empty map rather than just rely on the nil zero value?

Reply:
because we're going to be adding things to it, and I don't want to nilcheck before every add. zero maps panic when you attempt to add to them.